src/Stratis.Bitcoin.Features.BlockStore/BlockRepository.cs

```diff
@@ -33,9 +33,9 @@ public interface IBlockRepository : IBlockStore
         Task<List<Block>> GetBlocksAsync(List<uint256> hashes);
 
         /// <summary>
-        /// Wipe our blocks and their transactions then replace with a new block.
+        /// Wipe out blocks and their transactions then replace with a new block.
         /// </summary>
-        /// <param name="newTip">Hash and height of the new repository's tip.</param>
+        /// <param name="newTip">New repository's tip.</param>
```

Comment:
I actually meant to use what you did have here in that other method comment as well not the otherwise